tests/puppeteer/campaigns/functional/BO/catalog/brandsAndSuppliers/brands/01_filterAndQuickEditBrands.js

```diff
@@ -0,0 +1,180 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+const {demoBrands} = require('@data/demo/brands');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const BrandsPage = require('@pages/BO/brands');
+
+let browser;
+let page;
+let numberOfBrands = 0;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    brandsPage: new BrandsPage(page),
+  };
+};
+
+// Filter And Quick Edit brands
+describe('Filter And Quick Edit brands', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+  // Login into BO and go to brands page
+  loginCommon.loginBO();
+  it('should go to brands page', async function () {
```

Comment:
Maybe start to comment all the `it()` like we said we should ?

Reply:
:+1: